src/js/components/TextInput/TextInput.js

```diff
@@ -364,7 +368,10 @@ const TextInput = forwardRef(
                       ref={itemRef}
                     >
                       <Button
-                        active={activeSuggestionIndex === index}
+                        id={`${id}-listbox-option-${index}`}
+                        role="option"
+                        aria-selected={selected ? 'true' : undefined}
```

Comment:
Can we update aria-selected to just be true or false instead of true or undefined? Setting aria-selected to undefined means the option is not selectable (see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected#values)
```suggestion
                        aria-selected={selected}
```

Reply:
Sure. I copied this from the WAI-ARIA example but hadn't seen that note on MDN.

I think `aria-selected` must be a string, not a boolean attribute though.

Reply:
I've updated the options to add `aria-selected='false'` as the default state.